File: 1598. 文件夹操作日志搜集器/1598. 文件夹操作日志搜集器.java
class Solution {
    public int minOperations(String[] logs) {
        int level=0;
        for(String log:logs){
            if ("../".equals(log)) level=Math.max(level-1,0);
            else if (!"./".equals(log)) level--;
        }
        return level;
    }
}

File: 1598. 文件夹操作日志搜集器/1598. 文件夹操作日志搜集器.cpp
class Solution {
public:
    int minOperations(vector<string>& logs) {
        int level = 0;
        for(int i=0;i<logs.size();i++){
            if (logs[i]=="../") level=max(level-1,0);
            else if (logs[i]!="./") level++;
        }
        return level;
    }
};